package.json

```diff
@@ -23,7 +23,7 @@
   },
   "devDependencies": {},
   "dependencies": {
-    "assemblyscript": "github:nearprotocol/assemblyscript#assemblyscript-update",
+    "assemblyscript": "github:willemneal/assemblyscript#near_dist",
```

Comment:
Please make branch in nearprotocol repo, I've added you as collaborator.

Reply:
We should setup CI for external contributors as well. Gitlab CI does not work for that

Reply:
@bowenwang1996 what does work? Travis?

Reply:
Assemblyscript uses travis and it works